Tasks/UsePythonVersion/Tests/L0.ts

```diff
@@ -119,14 +121,52 @@ describe('UsePythonVersion L0 Suite', function () {
             const expectedMessage = [
                 'loc_mock_VersionNotFound 3.x',
                 'loc_mock_ListAvailableVersions',
-                '2.7.13'
+                '2.6.0 (x86)',
+                '2.7.13 (x86)',
+                '2.6.0 (x64)',
+                '2.7.13 (x64)'
             ].join(EOL);
 
             assert.strictEqual(e.message, expectedMessage);
             done();
         }
     });
 
+    it('selects architecture passed as input', async function () {
+        let buildVariables: { [key: string]: string } = {};
```

Comment:
It looks like you can make this `const`.

Reply:
Yeah, I've come around to your opinion on this.  I'll change this next time I touch this file.